.pylintrc

```diff
@@ -60,97 +60,24 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use "--disable=all --enable=classes
 # --disable=W".
-disable=apply-builtin,
```

Comment:
Those all are now deprecated options, and pylint warns they are gone and do nothing in the config file